renderer/components/Pay/PayPanelFooter.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box } from 'rebass'
+import { FormattedMessage } from 'react-intl'
+import { CryptoValue } from 'containers/UI'
+import { Message, Text } from 'components/UI'
+import messages from './messages'
+import PayButtons from './PayButtons'
+import { PAY_FORM_STEPS } from './constants'
+
+const isEnoughFunds = props => {
+  // convert entered amount to satoshis
+  const { amountInSats, channelBalance, invoice, isLn, isOnchain, walletBalanceConfirmed } = props
+
+  // Determine whether we have enough funds available.
+  let hasEnoughFunds = true
+  if (isLn && invoice) {
+    hasEnoughFunds = amountInSats <= channelBalance
+  } else if (isOnchain) {
+    hasEnoughFunds = amountInSats <= walletBalanceConfirmed
+  }
+
+  return hasEnoughFunds
+}
+
+const nextButtonText = (formState, props) => {
```

Comment:
can we rename this to `getNextButtonText` ? there is a local variable with the same name and in general `getX` is probably a better name for this kind of routine